lm_eval/tasks/gpqa/generative/_gpqa_generative_n_shot_yaml

```diff
@@ -0,0 +1,31 @@
+dataset_path: Idavidrein/gpqa
+group: gpqa
+output_type: generate_until
+process_docs: !function utils.process_docs
+training_split: train
+# Because huggingface dataset only has train split
+validation_split: train
+test_split: null
+description: "Here are some example questions from experts. Answer the final question yourself, following the format of the previous questions exactly.\n"
+doc_to_text: "Question: {{Question}}\nChoices:\n(A) {{choice1}}\n(B) {{choice2}}\n(C) {{choice3}}\n(D) {{choice4}}\nAnswer:"
+doc_to_target: answer
+filter_list:
+  - name: "get-answer"
+    filter:
+      - function: "regex"
+        regex_pattern: "((?<=The answer is )(.*)(?=.)|(?<=the answer is )(.*)(?=.)|(?<=The answer: )(.*)(?=.)|(?<=The final answer: )(.*)(?=.))"
```

Comment:
Do we expect typical models not trained for CoT explicitly to output this format? this non-CoT variant is likely especially important to have flexible answer extraction for

Reply:
Thanks for your advice! The version I referred to earlier was older, so I added flexible match to my latest commit and tested it.